electrumsv/gui/qt/network_dialog.py

```diff
@@ -1318,9 +1264,10 @@ def update_list(self, items: List[ServerListEntry]) -> None:
 
             is_connected = False
             considered_good = False
-            if list_entry.data_electrumx is not None:
-                is_connected = self._is_server_healthy(list_entry.data_electrumx,
-                    self._network.sessions)
+            # TODO (1.4.0) Maybe this is relevant to non ElectrumX server types?
```

Comment:
These should all be TODO(1.4.0) so they match reliably when searched for. Additionally, either this is or it is not, this should not be checked in which defers it to someone else, it should be resolved and removed.